### PR TITLE
Add Deploy on Koyeb guide

### PR DESCRIPTION
**Pull request by edouardb**
Hi there 👋,

Added a new guide to deploy Meilisearch on Koyeb.

### CODE REVIEW

File: learn/cookbooks/koyeb.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Deploy Meilisearch on Koyeb
+
+## Introduction
+
+This guide explains how to deploy a ready-to-use Meilisearch service on Koyeb.
```
Comment (dichotommy) [inline learn/cookbooks/koyeb.md:5]:
```suggestion
This guide explains how to deploy a ready-to-use Meilisearch instance on Koyeb.
```

File: learn/cookbooks/koyeb.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+[Koyeb](https://www.koyeb.com) is a developer-friendly serverless platform to deploy apps globally. The platform lets you seamlessly run Docker containers, web apps, and APIs with git-based deployment, TLS encryption, native autoscaling, a global edge network, and built-in service mesh & discovery.
+
+## Requirements
+
+To successfully follow and complete this guide, you need a [Koyeb account](https://app.koyeb.com).
+
+## Deploy Meilisearch
+
+The fastest way to deploy Meilisearch on Koyeb is to use the **Deploy to Koyeb** button to deploy. It allows you to have your Meilisearch instance running with minimal configuration.
```
Comment (dichotommy) [inline learn/cookbooks/koyeb.md:15]:
```suggestion
The fastest way to deploy Meilisearch on Koyeb is to use the **Deploy to Koyeb** button.
```

File: learn/cookbooks/koyeb.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+```bash
+python -c 'import os,base64; print(base64.urlsafe_b64encode(os.urandom(32)).decode())'
+```
+
+::: warning
+Setting up a master key is optional but strongly recommended when running in production. Unsetting the master key will make your Meilisearch instance unprotected and publicly accessible.
+:::
+
+### Test Meilisearch.
```
Comment (dichotommy) [inline learn/cookbooks/koyeb.md:31]:
```suggestion
### Test Meilisearch
```

File: learn/cookbooks/koyeb.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Copy the public URL (e.g. `https://meili-myorg.koyeb.app`) from the [Koyeb control panel](https://app.koyeb.com) and paste it your browser.
```
Comment (dichotommy) [inline learn/cookbooks/koyeb.md:33]:
```suggestion
Copy the public URL (e.g. `https://meili-myorg.koyeb.app`) from the [Koyeb control panel](https://app.koyeb.com) and paste it in your browser.
```

File: learn/cookbooks/koyeb.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+You land on the MeiliSearch dashboard, where you are asked to authenticate using your master key.
```
Comment (dichotommy) [inline learn/cookbooks/koyeb.md:35]:
```suggestion
You should land on the Meilisearch [search preview](/learn/what_is_meilisearch/search_preview.md), where you are asked to enter your master key.
```

File: learn/cookbooks/koyeb.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+You are now ready to create your first index!
```
Comment (dichotommy) [inline learn/cookbooks/koyeb.md:37]:
```suggestion
You are now ready to [create your first index](/learn/getting_started/quick_start.md)!
```

**Review by dichotommy — CHANGES_REQUESTED** [review]
Hi @edouardb , thanks for your PR! 👋🏻 

We are concerned that your "Deploy to Koyeb" button may not be visible in dark mode, which was recently added to our site. Could you pull the latest changes from master and have a look?

File: index.md [hunk 7]
```diff
@@ -15,6 +15,7 @@ Welcome to the Meilisearch documentation. Here you'll find everything you need t
 - [Deploy Meilisearch on AWS](/learn/cookbooks/aws.md)
 - [Deploy Meilisearch on GCP](/learn/cookbooks/gcp.md)
 - [Deploy Meilisearch on DigitalOcean](/learn/cookbooks/digitalocean_droplet.md)
+- [Deploy Meilisearch on Koyeb](/learn/cookbooks/koyeb.md)
```
Comment (dichotommy) [inline index.md:18]: Hi @edouardb ! Do you think you could also remove this line when you update this PR?

The deploy links on our homepage are only intended to cover the top 3 or 4 industry-standard cloud providers. Adding Koyeb to that list might mean adding others in the future, and that could make our homepage very crowded. I hope you understand—your guide will still be accessible from the sidebar in the Learn section of the site.

Thank you very much!

```suggestion
```

**Comment by edouardb** [comment]
> Hi @edouardb , thanks for your PR! 😄
> 
> We are concerned that your "Deploy to Koyeb" button may not be visible in dark mode, which was recently added to our site. Could you pull the latest changes from master and have a look?

Hi @dichotommy,

Thanks for the review!
I just checked the look of the button, it seems to be ok 👍 

Screen Shot 2022-03-22 at 10 11 01 AM

**Pull request merged**